features/step_definitions/stepdefs.js

```diff
@@ -1518,12 +1518,18 @@ Then(
             inboxUrl.pathname,
             (call) => {
                 const body = JSON.parse(call.request.body);
-                return (
-                    body.type === activityType &&
-                    (object
-                        ? body.object.id === object.id
-                        : body.object.type === objectNameOrType)
-                );
+                if (body.type !== activityType) {
+                    return false;
+                }
+
+                if (object) {
+                    if (typeof body.object === 'string') {
```

Comment:
This is because `Delete` activities just use an id for the `object` rather than an expanded object (because it's been deleted)